### DISCUSSION TITLE
Pull data as compressed file

### DISCUSSION THREAD

**Discussion by dinakarmu** [oracle/python-oracledb#406] [Ideas]
Hi, using Oracledb is there a way to fetch data from Oracle into a Parquet file or any other industry standard compressed file? if not, is such a feature in the plan?

**Comment by cjbj** [comment]
Currently there is ongoing investigation into Apache Arrow support.

**Reply by dinakarmu** [reply]
Thank you! Any alternatives available right now to read data into a compressed file format?

**Reply by cjbj** [reply]
@dinakarmu see https://github.com/oracle/python-oracledb/issues/375. @anthony-tuininga has pushed code to github that will help

**Comment by cjbj** [comment]
See https://github.com/oracle/python-oracledb/blob/main/samples/dataframe_parquet_write.py 

This needs python-oracledb 3.0 or greater.
